robot_skills/src/robot_skills/robot.py

```diff
@@ -231,6 +265,11 @@ def seq_or_none(obj):
                 discarded_reasons.append((arm_name, "required trajectories failed"))
                 continue
 
+            # Force sensor availability
```

Comment:
Ugly to add it this way, create an issue to get rid of this.

Reply:
issure #914